framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/widgets/RecentBlocks.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import * as React from 'react';
+import Widget from '../Widget';
+
+interface Props {
+	onClick?: (event: Event | React.MouseEvent<HTMLTableRowElement>) => void;
+}
+
+const RecentBlocks: React.FC<Props> = props => (
+	<Widget header={'Recent Blocks'}>
+		<table>
```

Comment:
if we dont create widget header and widget element, the styling of the header and row will be duplicated in each widget.
Also, from the design, i think header is not moving and only the row is moving?

Reply:
The sticky table header part should be implemented in the different table component not in the widget. Widget provide a simple wrapper and styling to put any thing in the widget.

Reply:
Related to styling of widget header, no it will not be duplicated. All widgets will refer to same header style. What is inside header is dynamic and user can add any component and its relevant styling depends on that custom header user use.

Reply:
> The sticky table header part should be implemented in the different table component not in the widget. 

maybe this was the misunderstanding. this issue meant to create all related component, not just the base widget component. and i agree the base widget is fine to render whatever its given to child